rpcs3/rpcs3qt/mainwindow.cpp

```diff
@@ -0,0 +1,233 @@
+#ifdef QT_UI
```

Comment:
This is not include guard actually, this just disables rpcs3qt compilation in main build system. Because rpcs3qt surrently is standalone app and causes rpcs3 build errors like this https://travis-ci.org/RPCS3/rpcs3/jobs/198356325